scripts/build.js

```diff
@@ -95,6 +95,9 @@ function removeBuildDir(val) {
   packageJSON[key] = removeBuildDir(packageJSON[key]);
 });
 
+// TODO: testing, included in bundle, do not require installation
+delete packageJSON.dependencies['broadcast-channel'];
```

Comment:
let's move `broadcast-channel` to devDeps, and delete the `devDependencies` field here